redfish-core/lib/account_service.hpp

```diff
@@ -1663,7 +1663,7 @@ inline void
     nlohmann::json& json = asyncResp->res.jsonValue;
     json["@odata.id"] = "/redfish/v1/AccountService";
     json["@odata.type"] = "#AccountService."
-                          "v1_10_0.AccountService";
+                          "v1_15_0.AccountService";
```

Comment:
This would be better to be combined in a string
```
    json["@odata.type"] = "#AccountService.v1_15_0.AccountService";
```

Reply:
done